client/src/app/tabs/bpmn/__tests__/BpmnEditorSpec.js

```diff
@@ -1064,6 +1067,103 @@ describe('<BpmnEditor>', function() {
       expect(elementTemplatesLoaderStub.setTemplates).to.be.calledTwice;
     });
 
+
+    it('should apply default templates to unsaved diagram', function(done) {
+
+      // given
+      const modeler = new BpmnModeler();
+
+      const invokeSpy = sinon.spy(modeler, 'invoke');
+
+      const cache = new Cache();
+
+      cache.add('editor', {
+        cached: {
+          modeler
+        }
+      });
+
+      function onImport() {
+
+        // then
+        expect(invokeSpy).to.have.been.calledWith(applyDefaultTemplates);
```

Comment:
```suggestion
        try {
          expect(invokeSpy).to.have.been.calledWith(applyDefaultTemplates);
        } catch (error) {
          return done(error);
        }
```
This will allow to avoid `timeout`ed tests in case feature is broken.

Reply:
Makes sense 👍